src/main/resources/graphics/scenery/volumes/AccumulateVDI.comp

```diff
@@ -0,0 +1,352 @@
+if (vis && step > localNear && step < localFar)
```

Comment:
This needs cleanup, there's a lot of commented code

Reply:
ab67bbc85227831d7a2df7fa94e1335522e256ec cleans up the code